server/auth_server.go

```diff
@@ -487,14 +484,16 @@ func RefreshToken(json map[string]string) (model.Token, error) {
 		}
 
 		payload := make(map[string]interface{})
+		if provider.GetName() == "ldap" {
+			payload["external_id"] = token.ExternalAccountID
+		}
```

Comment:
@prachidamle It is needed for token creation. Since ldap has no accessToken the payload would be passed empty without the `external_id` field